.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,17 @@
+name: Build and Deploy VTM App to Prod
+run-name: "${{ github.actor }} - ${{ github.ref_name }} - ${{ github.run_id }} 🚀"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  vtm-app:
+    uses: ./.github/workflows/deploy.yml
+    with:
+      branch: 'master'
```

Comment:
what if we wanted to deploy from a release branch?

Reply:
Current Jenkins VTM deploys are always from master. Would make sense to be releasing from a release branch, but i think we want to have a wider discussion about release process, as that break the current pattern.